### DISCUSSION TITLE
Zapdos Tutorial PDF - Eq 3.4: Grad Flux dot normal?

### DISCUSSION THREAD

**Discussion by ghost** [shannon-lab/zapdos#116] [Q&A Getting Started] [answered]
I'm looking through the Zapdos_Tutorial.pdf to cover my fundamentals, and I notice in equation 3.4 that the term labeled as "BCs" is written as:
<img width="163" alt="Screen Shot 2022-03-12 at 1 21 01 PM" src="https://user-images.githubusercontent.com/79124140/158031944-9c021dda-ef75-4523-99d3-5048537debda.png">
Shouldn't the del operator be absent from this term? It makes no sense to take the gradient of a vector.

Additionally, just as a check, the Advection term has a minus sign out in front:
<img width="212" alt="Screen Shot 2022-03-12 at 1 24 11 PM" src="https://user-images.githubusercontent.com/79124140/158032058-592cf86d-5a81-45cb-b537-cc287b854e8e.png">
I tried to derive 3.4 from 3.1 and 3.3 and I got a positive sign in front for the general case where the sign of the particle's charge is not specified, but how it it is written seems correct for the case of an electron. Did I go somewhere wrong in my derivation or is the minus sign intentional to highlight the electron case?

Thanks!

**Comment by csdechant** [comment, marked as answer]
No, I don't believe you make any mistakes. Those just seem to be some typos for both cases. I will make a note and edit the file.  Please let us know if you see anything else that doesn't make sense, or think of any way we can improve the documentation.
